docs/whats_new/version_ongoing.rst

```diff
@@ -29,6 +29,12 @@ Agnostic CCSocketCan
 ^^^^^^^^^^^^^^^^^^^^
 Incompatibilities with the agnostic proxy are now resolved. You should be able to use it again.
 
+Tester Present Sender
+^^^^^^^^^^^^^^^^^^^^^
+Add a context manager, tester present sender, to keep a UDS session open for
```

Comment:
Maybe just add the info what it does.

"Send cyclic tester present frames to keep UDS session alive..